source/process/feature-release.md

```diff
@@ -267,8 +267,8 @@ The final release is cut - RC cuts and bug fixes should be completed by this dat
 
 1. Release Manager:
     - Post this checklist in Release Checklist channel
-    - Verify all items in the last posted release checklist are complete, if not alert the release manager
-    - Schedule a release retrospective meeting, to be held within 5 days from the release
+    - Verify all items in the last posted release checklist are complete, if not alert the Release Manager
```

Comment:
```suggestion
    - Verify all items in the last posted release checklist are complete, if not alert the Release Manager
```
This is under the Release Manager's tasks. Confirming that they'd inform the Release Manager?

Reply:
It can just say "Verify all items in the last posted release checklist are complete"